types/fetch.d.ts

```diff
@@ -146,7 +146,8 @@ export declare class Request implements BodyMixin {
   readonly method: string
   readonly mode: RequestMode
   readonly redirect: RequestRedirect
-  readonly referrerPolicy: string
+  readonly referrer: string
+  readonly referrerPolicy: ReferrerPolicy
```

Comment:
Also narrowing down the referrer policy as we already have the `ReferrerPolicy` type used for `RequestInit`. It cannot really be anything else, to my best knowledge.